dali/operators/decoder/video/video_decoder_cpu.cc

```diff
@@ -12,66 +12,87 @@
+- C is the number of color channels
+
+.. code-block:: python
+
+    video_decoder = dali.experimental.decoders.Video(
+        frames=[0, 10, 20, 30, 40, 50, 40, 30, 20, 10, 0]
+    )
+
+Example 2: Extract a sequence of frames:
+
+.. code-block:: python
 
-The video streams can be in most of the container file formats. FFmpeg is used to parse video
- containers and returns a batch of sequences of frames with shape (F, H, W, C) where F is the
- number of frames in a sequence and can differ for each sample.)code")
+    video_decoder = dali.experimental.decoders.Video(
+        start_frame=0, sequence_length=10, stride=2
+    )
+
+Example 3: Pad the sequence by repeating the last frame:
+
+.. code-block:: python
+
+    video_decoder = dali.experimental.decoders.Video(
+        start_frame=0, sequence_length=100, stride=2, pad_mode="edge"
+    )
+
+)code")
     .NumInput(1)
     .NumOutput(1)
-    .InputDox(0, "buffer", "TensorList", "Data buffer with a loaded video file.")
+    .InputDox(0, "encoded", "TensorList", "Encoded video stream")
     .AddOptionalArg("affine",
-    R"code(Applies only to the mixed backend type.
+    R"code(Whether to pin threads to CPU cores (mixed backend only).
+
+If True, each thread in the internal thread pool will be pinned to a specific CPU core.
+If False, threads can migrate between cores based on OS scheduling.)code", true)
+    .AddOptionalArg<std::vector<int>>("frames",
+    R"code(Specifies which frames to extract from each video by their indices.
+
+The indices can be provided in any order and can include duplicates. For example, [0,10,5,10] would extract:
+- Frame 0 (first frame)
+- Frame 10
+- Frame 5 
+- Frame 10 (again)
+
+This argument cannot be used together with ``start_frame``, ``sequence_length``, ``stride`` or ``pad_mode``.)code", nullptr, true)
+    .AddOptionalArg<int>("start_frame",
+    R"code(Index of the first frame to extract from each video. Cannot be used together with frames argument.)code", nullptr, true)
+    .AddOptionalArg<int>("stride",
+    R"code(Number of frames to skip between each extracted frame. Cannot be used together with ``frames`` argument.)code", nullptr, true)
+    .AddOptionalArg<int>("sequence_length",
+    R"code(Number of frames to extract from each video. If not provided, the whole video is decoded. Cannot be used together with ``frames`` argument.)code", nullptr, true)
+    .AddOptionalArg<std::string>("pad_mode",
+    R"code(How to handle videos with insufficient frames when using start_frame/sequence_length/stride:
+- 'none': Return shorter sequences if not enough frames: ABC -> ABC
+- 'constant': Pad with a fixed value (specified by ``pad_value``): ABC -> ABCPPP
+- 'edge': Repeat the last valid frame: ABC -> ABCCCC
+- 'reflect_1001' or 'symmetric': Reflect padding, including the last element: ABC -> ABCCBA
+- 'reflect_101' or 'reflect': Reflect padding, not including the last element: ABC -> ABCBA
+Not relevant when using frames argument.)code", "constant", true)
```

Comment:
This argument renders in a weird way
![image](https://github.com/user-attachments/assets/b28a0e4f-acba-4a21-9e1e-530b9f4adb55)